_component_design/breadcrumbs.md

```diff
@@ -4,11 +4,11 @@ parent: Navigation
 layout: component
 category: Components
 usage: >
-  Breadcrumbs are an inline visual representation of page relationships and hierarchy. They provide a user with visual reinforcement of the current location within the application hierarchy, and allow for quick movement up the hierarchy.
+  Breadcrumbs are a linear navigation scheme that shows the users' current location in a website or application. A user can click  breadcrumb elements to navigate up the site hierarchy to previously visited pages.
```

Comment:
" users' "

Reply:
BCs are a hierarchical representation of the users' current location. They are critical for wayfinding and navigation up the hierarchical chain.

Reply:
Done.